loopdb/swapstate.go

```diff
@@ -110,7 +115,7 @@ func (s SwapState) Type() SwapStateType {
 func (s SwapState) IsPending() bool {
 	return s == StateInitiated || s == StateHtlcPublished ||
 		s == StatePreimageRevealed || s == StateFailTemporary ||
-		s == StateInvoiceSettled
+		s == StateInvoiceSettled || s == StateFailIncorrectHtlcAmt
```

Comment:
This state is declared 'pending' so that after a restart we pick up the swap again and check if it timed out.

Reply:
from a stale commit